projects/core/styles/components/button.less

```diff
@@ -35,6 +35,10 @@
     font: var(--tui-font-text-m);
     font-weight: bold;
 
+    &:disabled {
```

Comment:
Buttons have basic appearance applied to them, which already takes care of that:
https://github.com/taiga-family/taiga-ui/blob/main/projects/core/styles/components/appearance.less#L35